examples/installation/build.sh

```diff
@@ -135,14 +140,18 @@ xctest() {
         xcodebuild "${project[@]}" "${scheme[@]}" test "${destination[@]}" "${code_signing_flags[@]}"
     fi
 
-    if [[ $PLATFORM != osx ]]; then
+    if [ -d "$workspace" ]; then
+        [[ $LANG == 'swift' ]] && scheme=(-scheme RealmSwift) || scheme=(-scheme Realm)
+    else
+        scheme=()
+    fi
+
+    if [[ $PLATFORM != osx ]] && [[ $PLATFORM != catalyst ]]; then
         [[ $PLATFORM == 'ios' ]] && SDK=iphoneos || SDK=$PLATFORM
-        if [ -d "$workspace" ]; then
-            [[ $LANG == 'swift' ]] && scheme=(-scheme RealmSwift) || scheme=(-scheme Realm)
-        else
-            scheme=()
-        fi
         xcodebuild "${project[@]}" "${scheme[@]}" -sdk "$SDK" build "${code_signing_flags[@]}"
+    elif [[ $PLATFORM == catalyst ]]; then
```

Comment:
Moving this above the other case so that it's "if catalyst, else if not osx" rather than "if not osx and not catalyst, else if catalyst" makes the conditionals a little less weird.